tasks/service-runner/config/query-cmr-sidecar.yaml

```diff
@@ -97,4 +97,4 @@ spec:
       protocol: TCP
   selector:
     name: query-cmr
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
Made this change because 'LoadBalancer' could result in the services being exposed outside the cluster which is not needed and could cause problems later with local development on Docker Desktop.